src/main/java/uk/gov/hmcts/reform/profilesync/oidc/JwtGrantedAuthoritiesConverter.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.profilesync.oidc;
+
+import static org.springframework.security.oauth2.core.endpoint.OAuth2ParameterNames.ACCESS_TOKEN;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+import uk.gov.hmcts.reform.profilesync.repository.IdamRepository;
+
+@Component
+public class JwtGrantedAuthoritiesConverter implements Converter<Jwt, Collection<GrantedAuthority>> {
```

Comment:
can you put some comment on classes to add the purpose and logic in that class.. since these are framework classes, it will help team in future

Reply:
ok. sure

Reply:
added comments